tools/twix/src/main.rs

```diff
@@ -379,6 +531,13 @@ impl App for TwixApp {
                 self.dock_state
                     .set_focused_node_and_surface((surface_index, node_id));
             }
+
+            if let Some((surface_index, node_id)) = self.dock_state.focused_leaf() {
+                let node = &self.dock_state[surface_index][node_id];
+                let rect = node.rect().unwrap();
+                ui.painter()
+                    .rect_stroke(rect, Rounding::same(4.0), Stroke::new(1.0, Color32::RED));
```

Comment:
Maybe sue `ui.style().visuals.widgets.active.fg_stroke` as the stroke here?

Reply:
Yes :heart_eyes: